packages/components/src/internal/components/entities/ParentEntityEditPanel.tsx

```diff
@@ -122,35 +122,40 @@ export class ParentEntityEditPanel extends Component<Props, State> {
         let parentTypeOptions = List<IEntityTypeOption>();
         let originalParents = List<EntityChoice>();
 
-        await Promise.all(
-            parentDataTypes.map(async parentDataType => {
-                try {
-                    const typeData = await getParentTypeDataForSample(
-                        parentDataType,
-                        childData ? [childData] : [],
-                        childContainerPath
-                    );
-                    parentTypeOptions = parentTypeOptions.concat(typeData.parentTypeOptions) as List<IEntityTypeOption>;
-                    originalParents = originalParents.concat(
-                        getInitialParentChoices(
-                            typeData.parentTypeOptions,
+        try {
```

Comment:
curious why this outer try catch is needed, since each individual promise is already in try cach

Reply:
Yeah, it's not.  Just me trying to track down a missing `try-catch` block that the tests were highlighting.